grub.d/05_zfs_linux.py

```diff
@@ -100,11 +100,11 @@ def __init__(self, linux: str,
         except RuntimeError as e:
             sys.exit(e)
         self.version = self.get_linux_version()
-
+        
```

Comment:
A bunch of these unnecessary white space changes were made, could you remove them?

Reply:
I removed them and formatted the code with `autopep8 --max-line-length=100`.